test/book_spec.js

```diff
@@ -20,23 +20,37 @@ describe('book', function() {
     });
 });
 
-describe('sortBooks', function (done) {
+describe('reading books from the database', function(done) {
     // seed models in DB
     beforeEach(() => { 
         const reading =  new Book({ title: 'Book 1', currentlyReading: true })
         const read =  new Book({ title: 'Book 2', readOn: new Date() })
-        const tbr =  new Book({ title: 'Book 3' })
+        const tbr1 =  new Book({ title: 'Book 3' })
+        const tbr2 =  new Book({ title: 'Book 4' })
 
         reading.save() 
             .then(() => read.save())
-            .then(() => tbr.save())
+            .then(() => tb1r.save())
+            .then(() => tb2r.save())
             .then(() => done()); 
     }); 
-    it('should sort books into read, currentlyReading, and toBeRead', async function () {
-        sortBooks().then(() => {
-            expect(sortedBooks.reading[0].title).to.equal('Book 1')
-            expect(sortedBooks.read[0].title).to.equal('Book 2');
-            expect(sortedBooks.toBeRead[0].title).to.equal('Book 3');
-        })
+
+    describe('toBeRead', function (done) {
+        it('should return books that are not read and not currentlyReading', async function () {
+            toBeRead().then((books) => {
+                expect(books.length).to.equal(1);
+                expect([tbr1.title, tbr2.title]).to.include(books[0].title);
+            })
```

Comment:
i see you using a lot of promises in your code even though you are in async functions.  generally you use `await` in async functions so this would look like:
```
await toBeRead();
expect(books.length).to.equal(1);
expect([tbr1.title, tbr2.title]).to.include(books[0].title);
```